### ISSUE TITLE
pythreejs not showing without manually-specified scene translation

### ISSUE THREAD

**Issue by bdice** [glotzerlab/plato#37]
I am having trouble with getting blank scenes unless I manually specify the scene translation. I'm running commit c42e94310d742f0e60e8f874ff9aad548156e842 (current master, after v1.5.0).

To reproduce:
```python
import plato
import plato.draw.pythreejs as draw

# This is empty
draw.Scene(
    draw.Spheres(positions=[[0, 0, 0]], radii=[0.5], colors=[[0.5, 0.5, 0.5, 1]])).show()

# This shows the sphere
draw.Scene(
    draw.Spheres(positions=[[0, 0, 0]], radii=[0.5], colors=[[0.5, 0.5, 0.5, 1]]),
    translation=[0, 0, 0]).show()
```

**Comment by klarh** [comment]
This is probably due to interactions with the default placement/settings of pythreejs objects and the coupling of the camera and scene translation/rotation. Can you see if the `fix_pythreejs_translation` branch fixes the more complex use case you had?

**Comment by klarh** [comment]
Presumed fixed in f81a7fe, let me know if that's not the case.

**Comment by bdice** [comment]
@klarh Sorry, I didn't reply! I tested it earlier, that fixed the issue. Thanks! 😄
